From 92b65d8b75925226b38bd6efbb049bb72af887f8 Mon Sep 17 00:00:00 2001
From: Brooke Tilley <brooke.tilley@flexibits.com>
Date: Wed, 19 Jun 2024 14:40:58 -0400
Subject: [PATCH] [FAN-7003] hopefully fix stdalign.h missing error on CI

---
 icu4c/source/test/cintltst/cintltst.vcxproj | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/icu4c/source/test/cintltst/cintltst.vcxproj b/icu4c/source/test/cintltst/cintltst.vcxproj
index 9f3b8129e3..38e01e7955 100644
--- a/icu4c/source/test/cintltst/cintltst.vcxproj
+++ b/icu4c/source/test/cintltst/cintltst.vcxproj
@@ -32,6 +32,9 @@
       <TypeLibraryName>$(OutDir)\cintltst.tlb</TypeLibraryName>
     </Midl>
     <ClCompile>
+      <LanguageStandard>stdcpp17</LanguageStandard>
+      <LanguageStandard_C>stdc11</LanguageStandard_C>
+      <DisableLanguageExtensions>false</DisableLanguageExtensions>
       <WarningLevel>Level3</WarningLevel>
       <CompileAs>Default</CompileAs>
       <AdditionalIncludeDirectories>..\..\..\include;..\..\tools\ctestfw;..\..\common;..\..\i18n;..\..\tools\toolutil;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-- 
2.42.0.windows.2

